### PR TITLE
Issue 2262

### PR DESCRIPTION
**Pull request by bhasin85**

### CODE REVIEW

**Comment by pmbohm** [comment]
@bhasin85 tests broken?

**Comment by bhasin85** [comment]
Thanks @pmbohm  Looking into it.

File: web-app/js/portal/cart/DownloadEstimator.js [hunk 1]
```diff
@@ -119,5 +128,29 @@ Portal.cart.DownloadEstimator = Ext.extend(Object, {
...
+    },
+
+    _spatialSubsetIntersect: function(collection) {
+
+        var intersect = true;
+        var filters = collection.getFilters();
+        var params = filters.filter(function(filter) {
+            return filter.isNcwmsParams;
+        })[0];
+        var bounds = collection.getBounds();
```
Comment (pmbohm) [inline web-app/js/portal/cart/DownloadEstimator.js]: As the bounds is an Openlayers bounds you can use Openlayers functions to determine if 'rectangles intercect'
http://dev.openlayers.org/docs/files/OpenLayers/BaseTypes/Bounds-js.html#OpenLayers.Bounds.containsBounds

File: web-app/js/portal/cart/DownloadEstimator.js [hunk 2]
```diff
@@ -137,9 +137,9 @@ Portal.cart.DownloadEstimator = Ext.extend(Object, {
         var params = filters.filter(function(filter) {
             return filter.isNcwmsParams;
         })[0];
-        var bounds = collection.getBounds();
 
-        if (bounds && params && params.latitudeRangeStart != undefined) {
```
Comment (pmbohm) [inline web-app/js/portal/cart/DownloadEstimator.js]: What if params is null?
`if (bounds && params && params.latitudeRangeStart != undefined) {` 
is safer

Comment (pmbohm) [inline web-app/js/portal/cart/DownloadEstimator.js]: OK its fine . my bad

**Comment by pmbohm** [comment]
Should stop #2262 occuring

**Pull request merged**